README.md

```diff
@@ -14,57 +14,54 @@ Version 2.0 includes support for the core workflows of the following APIs:
 * [Orders](https://developers.planet.com/docs/orders/) - Process and download or deliver imagery.
 * [Subscriptions](https://developers.planet.com/docs/subscriptions/) - Set up a search to auto-process and deliver imagery.
 
-After the initial 2.0 release there will be additional work to support the 
+After the initial 2.0 release there will be additional work to support the
 remaining Planet APIs: [basemaps](https://developers.planet.com/docs/basemaps/),
-[tasking](https://developers.planet.com/docs/tasking/)) and 
+[tasking](https://developers.planet.com/docs/tasking/) and
 [analytics](https://developers.planet.com/docs/analytics/).
 
 ## Versions and Stability
 
-The default branch (main) of this repo is for the [Planet SDK for 
+The default branch (main) of this repo is for the [Planet SDK for
 Python](https://github.com/planetlabs/planet-client-python/projects/2),
-a complete rewrite and upgrade from the original [Planet Python 
-Client](https://developers.planet.com/docs/pythonclient/). If you 
-are looking for the source code to that library see the 
+a complete rewrite and upgrade from the original [Planet Python
+Client](https://developers.planet.com/docs/pythonclient/). If you
+are looking for the source code to that library see the
 [v1](https://github.com/planetlabs/planet-client-python/tree/v1) branch.
 
 The Planet SDK for Python is in 'pre-release' stages, working towards a solid
```

Comment:
While we're updating the README, this is no longer accurate. Could reword to just point to releases?

Reply:
Yup!  The entire versions/stability section has been rewritten.